.github/workflows/android-maestro.yml

```
@@ -87,32 +90,48 @@ jobs:
 
             -   name: Build workspace JS dependencies
+                timeout-minutes: 10
                 run: yarn build --filter=@rnw-community/react-native-payments
 
             -   name: Generate native project (expo)
                 if: matrix.target == 'expo'
+                timeout-minutes: 10
                 working-directory: ${{ env.EXAMPLE_DIR }}
                 run: yarn prebuild:expo
 
             -   name: Set up Android SDK
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)